### PR TITLE
doc/md/integrations: kubernetes operator docs

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/md/integrations/kubernetes-operator.mdx [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Background
+
+### What are Kubernetes Operators?
+
+For many stateful resources, such as reconciling the desired state of a database with its actual state
```
Comment (hilakashai) [inline doc/md/integrations/kubernetes-operator.mdx:14]:
```suggestion
For many stateful resources, reconciling the desired state of a database with its actual state
```

File: doc/md/integrations/kubernetes-operator.mdx [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+can be a complex task that requires a lot of domain knowledge. [Kubernetes Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+were introduced to the Kubernetes ecosystem to help users manage complex stateful resources by codifying
+this domain knowledge into a Kubernetes controller.
+
+### What is the Atlas Kubernetes Operator?
+
+The Atlas Kubernetes Operator is a Kubernetes controller that uses Atlas to manage
+the schema of your database. The Atlas Kubernetes Operator allows you to define the desired schema of your
```
Comment (hilakashai) [inline doc/md/integrations/kubernetes-operator.mdx:22]:
```suggestion
the schema of your database. The Atlas Kubernetes Operator allows you to define the desired schema of your
```
```suggestion
your database schema. The Atlas Kubernetes Operator allows you to define the desired schema 
```

File: doc/md/integrations/kubernetes-operator.mdx [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+The `diff` field is used to define the diff policies that are used to plan the schema changes.
+
+```yaml
+policy:
+  diff:
+   skip:
+    drop_column: true
+```
+
+Currently, the only diff policy available is `skip`. This policy controls which kind of schema chagnes will be
```
Comment (hilakashai) [inline doc/md/integrations/kubernetes-operator.mdx:274]:
```suggestion
Currently, the only diff policy available is `skip`. This policy controls which kind of schema changes will be
```

File: doc/md/integrations/kubernetes-operator.mdx [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+* Versioned schema migrations - coming soon. 
+
+### Declarative schema migrations
+
+![](https://user-images.githubusercontent.com/1522681/236139615-1d10feea-8b19-46a2-905b-b614883c48c8.png)
+
+The Atlas Kubernetes Operator supports [declarative migrations](https://atlasgo.io/concepts/declarative-vs-versioned#declarative-migrations).
+In declarative migrations, the desired state of the database is defined by the user and the operator is responsible
+for reconciling the desired state with the actual state of the database (planning and executing `CREATE`, `ALTER`
+and `DROP` statements).
```
Comment (masseelch) [inline doc/md/integrations/kubernetes-operator.mdx:37]: Maybe add another hint stating versioned migrations support will come soon.

File: doc/md/integrations/kubernetes-operator.mdx [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+
+## Getting started
+
+### Installation
+
+The Atlas Kubernetes Operator is available as a Helm chart. To install the chart with the release name `atlas-operator`:
```
Comment (masseelch) [inline doc/md/integrations/kubernetes-operator.mdx:43]:
```suggestion
The Atlas Kubernetes Operator is available as a Helm Chart. To install the chart with the release name `atlas-operator`:
```

image

File: doc/md/integrations/kubernetes-operator.mdx [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+
+```yaml title="db-credentials.yaml"
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-credentials
+type: Opaque
+stringData:
+  url: "mysql://user:pass@your.db.com:3306/"
+```
```
Comment (masseelch) [inline doc/md/integrations/kubernetes-operator.mdx:95]:
```suggestion
```
This defines a database-bound connection.
```

File: doc/md/integrations/kubernetes-operator.mdx [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+```yaml title="db-credentials.yaml"
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pg-credentials
+type: Opaque
+stringData:
+  url: "postgres://user:pass@your.db.com:5432/?search_path=myapp&sslmode=disable"
+```
+This defines a schema-bound connection to a schema called `myapp`.
```
Comment (masseelch) [inline doc/md/integrations/kubernetes-operator.mdx:109]: In default database (usually `postgres`)

**Review by masseelch — APPROVED** [review]
Some minor suggestions.

**Pull request merged**